### PR TITLE
feat: root components

### PR DESCRIPTION
**Pull request by Kojot999**

### CODE REVIEW

File: src/app/components/Panel/Filtres/Filters.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styles from "./Filters.module.scss";
+import sort from "../../../../Img/sort.svg";
+
+export const Filters = () => {
+  return (
+    <div className={styles.wrapper}>
+      <img alt="^" src={sort} />
```
Comment (pislagz) [inline src/app/components/Panel/Filtres/Filters.js:7]:
```suggestion
      <img alt="sort" src={sort} />
```

File: src/app/components/Panel/Search/Search.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styles from "./Search.module.scss";
+import search from "../../../../Img/star.svg";
+
+export const Search = () => {
+  return (
+    <div className={styles.wrapper}>
+      <img alt="" src={search} />
```
Comment (pislagz) [inline src/app/components/Panel/Search/Search.js:7]:
```suggestion
      <img alt="search" src={search} />
```

**Review by pislagz — COMMENTED** [review]
Only improve the `alt` tags, the rest is ok.

File: src/app/components/Panel/Sort/Sort.module.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    border: none;
+    border-radius: 10px;
+    color: #fff;
+    font-size: 10px;
+    white-space: nowrap;
+    cursor: pointer;
+    padding-inline: 10px;
+    &.isOpened {
+      border-bottom-left-radius: 0px;
+      border-bottom-right-radius: 0px;
```
Comment (pislagz) [inline src/app/components/Panel/Sort/Sort.module.scss:17]: Remove units when value is zero.

File: src/app/components/Panel/Sort/Sort.module.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    background-color: #23243b;
+    border-radius: 10px;
+    cursor: pointer;
+    ul {
+      list-style: none;
+      padding-inline: 5px;
+      margin-top: 10px;
+    }
+    &.isOpened {
+      border-top-left-radius: 0px;
```
Comment (pislagz) [inline src/app/components/Panel/Sort/Sort.module.scss:39]: Same here.

File: src/app/components/Tiles/HomeTile/HomeTile.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <div className={styles.wrapper}>
+      <div key={id} className={styles.tile}>
+        <div className={styles.wrapperImg}>
+          <h3 className={styles.title}>{name}</h3>
+          <img alt="" src={backgroundImage} className={styles.gameImg} />
+        </div>
+        <img alt="star" src={star} className={styles.starIcon} />
+
```
Comment (pislagz) [inline src/app/components/Tiles/HomeTile/HomeTile.js:23]: Redundant empty space.

File: src/app/views/Stores/Stores.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import styles from "./Stores.module.scss";
+import { useContent } from "./useContent";
+
+export const Stores = () => {
+  const { data, isLoading, error } = useContent();
+  const isContentVisible = !isLoading && !error;
+  const { stores } = data;
+
+  return (
+    <>
```
Comment (Sabian329) [inline src/app/views/Stores/Stores.js:11]: unnecessary react fragment

File: src/app/components/Panel/Filtres/Filters.js (hunk 1 above)
Comment (Kojot999) [reply src/app/components/Panel/Filtres/Filters.js]: Done

File: src/app/components/Panel/Sort/Sort.module.scss (hunk 3 above)
Comment (Kojot999) [reply src/app/components/Panel/Sort/Sort.module.scss:17]: Done

**Pull request merged**